### PR TITLE
(chore): Node.js default tests improvements

### PR DESCRIPTION
**Pull request by sebs**
*Description of changes:*

* 1 assertion per test. This is easier to digest and debug
* consistent usage of semicolons and string quotes
* body and result test split

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: samcli/local/init/templates/cookiecutter-aws-sam-hello-nodejs/{{cookiecutter.project_name}}/hello-world/tests/unit/test-handler.js [hunk 1]
```diff
@@ -6,17 +6,37 @@ const expect = chai.expect;
...
+        before(() => {
+            body = JSON.parse(result.body);
+        })
+        it('parses to object', () => {
+            expect(body).to.be.an('object');
+        });
+        it('message is hello world', () => {
+            expect(body.message).to.be.equal('hello world');
+        });
+        it('location is a string', () => {
```
Comment (TheSriram) [inline samcli/local/init/templates/cookiecutter-aws-sam-hello-nodejs/{{cookiecutter.project_name}}/hello-world/tests/unit/test-handler.js]: Can this line be commented out? just like the original example.

**Comment by TheSriram** [comment]
I can take over this PR and finish it up. @sebs do you mind?

**Comment by sebs** [comment]
No i dont mind, please do so as I am a irresponsible person here. Thank you ! <3

**Comment by TheSriram** [comment]
Great, i'll take this up and wrap it up next week.

**Comment by sriram-mv** [comment]
Closing this, as code change needs to move to : https://github.com/awslabs/aws-sam-cli-app-templates

**Pull request closed without merging**